src/TurbulenceClosures/turbulence_closure_implementations/CATKEVerticalDiffusivities/CATKEVerticalDiffusivities.jl

```diff
@@ -315,12 +349,13 @@ end
         on_bottom = !inactive_cell(i, j, k, grid) & inactive_cell(i, j, k-1, grid)
         Δz = Δzᶜᶜᶜ(i, j, k, grid)
         Cᵂϵ = closure_ij.turbulent_kinetic_energy_equation.Cᵂϵ
-        Q_e = - Cᵂϵ * turbulent_velocityᶜᶜᶜ(i, j, k, grid, closure_ij, tracers.e) / Δz * on_bottom
+        Q_e = - Cᵂϵ * w★[i, j, k] / Δz * on_bottom
```

Comment:
We should discuss whether it would be better to keep the design that uses the function `turbulent_velocity`. I tried to write this code so that we could experiment with putting TKE at cell interfaces. All of these changes hard code the TKE location, making it harder to modify in the future...